src/language-css/printer-postcss.js

```diff
@@ -402,7 +402,10 @@ function genericPrint(path, options, print) {
     }
     case "value-paren": {
       if (n.raws.before !== "") {
-        return concat([line, n.value]);
+        const parent = path.getParentNode(2);
+        const isFunction = parent.type === "value-func";
+
+        return concat([isFunction ? "" : line, n.value]);
```

Comment:
What happens if you have a function call as a parameter? Such as:

```scss
clip: rect(0,  round($card-score-size / 2), 0, 0)
```

Reply:
@lydell add to tests